sapi/cli/tests/php_cli_server.inc

```diff
@@ -29,7 +29,7 @@ function php_cli_server_start($code = 'echo "Hello world";', $router = 'index.ph
 		if (!is_null($router)) {
 			$cmd .= " {$router}";
 		}
-		$cmd .= " 2>/dev/null";
+		$cmd .= !$stdout_file && !$stderr_file ? " 2>/dev/null" : "";
```

Comment:
a explicit if feels more readable

Reply:
Sorry, not sure I understand what you mean. Do you mean doing something like this?

```php
$ignore_output = !$stdout_file && !$stderr_file;
$cmd .= $ignore_output ? " 2>/dev/null" : "";
```

Reply:
```php
if (!$stdout_file && !$stderr_file) {
    $cmd .= " 2>/dev/null":
}
```

Reply:
Gotcha. Updated :)